src/gpu/d3d11/SDL_gpu_d3d11.c

```diff
@@ -5630,12 +5630,12 @@ static bool D3D11_SupportsTextureFormat(
 
 // Device Creation
 
-static bool D3D11_PrepareDriver(SDL_VideoDevice *_this)
+static bool D3D11_PrepareDriver(SDL_VideoDevice *this)
 {
     void *d3d11_dll, *dxgi_dll;
-    PFN_D3D11_CREATE_DEVICE D3D11CreateDeviceFunc;
+    PFN_D3D11_CREATE_DEVICE d3D11CreateDeviceFunc;
```

Comment:
Something happened here with the casing. Thankfully outside of the function pointers I don't think there's too many other instances of variables beginning with a capital letter